wetterdienst/provider/dwd/observation/api.py

```diff
@@ -324,11 +324,16 @@ def _create_humanized_parameters_mapping(self) -> Dict[str, str]:
         """
         hpm = {}
 
-        for parameter in DwdObservationParameter[self.sr.resolution.name]:
-            try:
-                hpm[parameter.value] = parameter.name.lower()
-            except AttributeError:
-                pass
+        datasets = [
+            dataset for dataset in DwdObservationParameter[self.sr.resolution.name] if hasattr(dataset, "__name__")
+        ]
+
+        for dataset in datasets:
```

Comment:
Would not it easier to have a fixed mapping between parameter names and humanized names? While using rename you can use dicts with hundreds of entries.